frontend/src/js/components/devices/device-details/authsets/AuthSetList.test.tsx

```diff
@@ -32,9 +32,9 @@ MIIBoj
       status: 'accepted'
     };
     const authSets = Object.keys(DEVICE_STATES).reduce((accu, status, index) => {
-      accu.push({ ...authset, id: `${status}-${index}`, status, ts: `2020-09-21T12:${42 + index - 5}:34.571Z` });
-      accu.push({ ...authset, id: `${status}-${index}`, status, ts: `2020-09-21T12:${42 + index}:34.571Z` });
-      accu.push({ ...authset, id: `${status}-${index}`, status, ts: `2020-09-21T12:${42 + index + 5}:34.571Z` });
+      accu.push({ ...authset, id: `${status}-${index}-1`, status, ts: `2020-09-21T12:${42 + index - 5}:34.571Z` });
+      accu.push({ ...authset, id: `${status}-${index}-2`, status, ts: `2020-09-21T12:${42 + index}:34.571Z` });
+      accu.push({ ...authset, id: `${status}-${index}-3`, status, ts: `2020-09-21T12:${42 + index + 5}:34.571Z` });
```

Comment:
is the `index` here not unique already?

Reply:
The index is unique for each reduce iteration, but it's used three times per iteration, which is why it's causing an issue

Reply:
argh 🤦